docs/StardustDocs/topics/percentile.md

```diff
@@ -0,0 +1,32 @@
+[//]: # (title: percentile)
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Analyze-->
+
+Computes the specified percentile of values.
+
+Is available for `Comparable` columns. [`NA` values](nanAndNa.md#na) (`null` and `NaN`) are ignored.
+
+<!---FUN percentileModes-->
+
+```kotlin
+df.percentile(25.0) // percentile of values per every comparable column
+df.percentile(25.0) { age and weight } // percentile of all values in `age` and `weight`
+df.percentileFor(25.0) { age and weight } // percentile of values per `age` and `weight` separately
+df.percentileOf(25.0) { (weight ?: 0) / age } // percentile of expression evaluated for every row
+```
+
+<!---END-->
+
+<!---FUN percentileAggregations-->
+
+```kotlin
+df.percentile(25.0)
+df.age.percentile(25.0)
```

Comment:
is it possible to embed a table here, like in valueColumns https://kotlin.github.io/dataframe/valuecounts.html

Other statistics are poorly covered with examples, should be improved

Reply:
We could review all statistical functions documentation.